.gitpod.yml

```diff
@@ -18,7 +18,7 @@ tasks:
 
         yarn install
 
-        rails g webpacker:install && rails g webpacker:install:react && rails g react:install
+        rails webpacker:install && rails webpacker:install:react && rails g react:install
```

Comment:
```suggestion
```

Reply:
OK, i see this message in the output in GitPod:

```
    conflict  config/webpacker.yml
Overwrite /workspace/plots2/config/webpacker.yml? (enter "h" for help) [Ynaqdhm]
```

We could decide to move fwd with removing that entire line, and see what it does in GitPod?